common/constants.js

```diff
@@ -64,6 +64,7 @@ module.exports = Object.freeze({
     BACKUP_STATUS_POLLER: 'FabrikStatusPoller',
```

Comment:
we can remove this , right ?

Reply:
Supporting on going backups using FabrikStatusPoller. FabrikStatusPoller would poll status for on-going backups before rolling out BnRStatusPoller. FabrikStatusPoller can be remove anytime after 24 hrs of rolling out of BnRStatusPoller.